org.eclipse.xtext/src/org/eclipse/xtext/build/IncrementalBuilder.java

```diff
@@ -199,9 +205,16 @@ public URIBasedFileSystemAccess newFileSystemAccess(Resource resource, BuildRequ
 		private OperationCanceledManager operationCanceledManager;
 
 		protected void unloadResource(URI uri) {
+			unloadResource(uri, x -> true);
+		}
+
+		/**
+		 * @since 2.28
```

Comment:
Suggested JavaDoc:

Unload the resource with the given uri if it satisfies the given condition.

Reply:
done